src/main/resources/data/spectrum/loot_tables/chests/undergrowth_manor/basement_sinners_outcrop.json

```diff
@@ -0,0 +1,141 @@
+{
+  "type": "minecraft:chest",
+  "pools": [
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "spectrum:nightdew_sprout",
+          "functions": [
+            {
+              "function": "minecraft:set_count",
+              "count": {
+                "min": 20,
+                "max": 34
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rolls": {
+        "min": 2,
+        "max": 3
+      },
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:nether_wart",
+          "functions": [
+            {
+              "function": "minecraft:set_count",
+              "count": {
+                "min": 10,
+                "max": 30
+              }
+            }
+          ]
+        },
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:large_fern",
+          "functions": [
+            {
+              "function": "minecraft:set_count",
+              "count": {
+                "min": 12,
+                "max": 28
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "spectrum:nectardew_burgeon",
+          "weight": 666,
+          "quality": 666
```

Comment:
While humorous, I want to note that setting these values will cause the burgeons to be affected by luck. A single point of negative luck (maybe via Bad Luck) will cause this to be unfindable. 
Otherwise it should function as expected, unless something something mod compat shenanigans.

Reply:
Oh this is true, i just set these to arbitrarily high numbers to try to make sure that they don’t get overwritten by some other mod injecting into the loot table, combined with high luck
(looking at YOU simply swords)
the concern with unluck is… valid though.